docs/Utilities/SessionToken.md

```diff
@@ -46,11 +52,14 @@ function App() {
   return <TextField label="Session Token" value={token} disabled />;
 }
 
-render(ExtensionPoint.MyExtension, () => <App />);
+extend(
+  'Admin::Product::SubscriptionPlan::Add',
+  render(() => <App />),
+);
 ```
 
 ## Session Token API
 
-| Name            | Type                                | Description                | Required |
-| --------------- | ----------------------------------- | -------------------------- | -------- |
-| getSessionToken | `() => Promise<string | undefined>` | Get a fresh session token. | ☑️       |
+| Name            | Type                                 | Description                | Required |
+| --------------- | ------------------------------------ | -------------------------- | -------- |
+| getSessionToken | `() => Promise<string \| undefined>` | Get a fresh session token. | ☑️       |
```

Comment:
```suggestion
| getSessionToken | `() => Promise<string | undefined>` | Get a fresh session token. | ☑️       |
```

Reply:
This back slash escapes the | or else it'll mistaken it for table cell separator on markdown.

Reply:
I see...! I guess my IDE's .md preview doesn't have the same issue. Good to know.